awx/ui/client/src/templates/job_templates/job-template.form.js

```diff
@@ -43,6 +43,19 @@ function(NotificationsList, i18n) {
                     column: 1,
                     ngDisabled: '!(job_template_obj.summary_fields.user_capabilities.edit || canAddJobTemplate)'
                 },
+                organization: {
+                    label: i18n._('Organization'),
+                    type: 'lookup',
+                    list: 'OrganizationList',
+                    sourceModel: 'organization',
+                    basePath: 'organizations',
+                    sourceField: 'name',
+                    dataTitle: i18n._('Organization'),
+                    dataContainer: 'body',
+                    dataPlacement: 'right',
+                    ngDisabled: '!(job_template_obj.summary_fields.user_capabilities.edit || canAddJobTemplate)',
+                    awLookupWhen: '(job_template_obj.summary_fields.user_capabilities.edit || canAddJobTemplate)'
+                },
```

Comment:
oh this probably isn't right anymore

Reply:
@AlanCoding Are you referring to the user capabilities summary field?

Reply:
Given that it is supposed to be read-only and implicitly inherited from the project relationship, should this field _always_ be disabled?